pkg/controllers/csi/metadata/migrations.go

```diff
@@ -0,0 +1,93 @@
+package metadata
+
+import (
+	"path/filepath"
+
+	dtcsi "github.com/Dynatrace/dynatrace-operator/pkg/controllers/csi"
+	"gorm.io/gorm"
+)
+
+func dataMigration(tx *gorm.DB) error {
+	var dynakubes []Dynakube
+
+	tx.Table("dynakubes").Find(&dynakubes)
+
+	for _, d := range dynakubes {
+		tc := TenantConfig{
+			Name:                        d.Name,
+			TenantUUID:                  d.TenantUUID,
+			ConfigDirPath:               filepath.Join(filepath.Join(dtcsi.DataPath, d.TenantUUID), d.Name, dtcsi.SharedAgentConfigDir),
```

Comment:
why not use the `PathResolver`, to remain consistent with the rest of the code? (its even in the same package 😅 )

Reply:
Because it's unclear how to creat an instance of it... but I can do it.

Reply:
you can create a constructor for it, or just `PathResolver{RootDir: dtcsi.DataPath}`

Reply:
fixed 23abf8ea